contracts/GSN/GSNRecipientERC20Fee.sol

```diff
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: MIT
 
-pragma solidity >=0.6.0 <0.8.0;
+pragma solidity >=0.6.0 <0.9.0;
 
 import "./GSNRecipient.sol";
 import "../math/SafeMath.sol";
```

Comment:
SafeMath could now be deprecated, right?

Reply:
In practice, yes. But I would not remove it here since it's a huge part of the `@openzeppelin/contracts` package and a lot of projects still depend on it.